aptos-move/framework/cached-packages/src/aptos_framework_sdk_builder.rs

```diff
@@ -663,6 +669,21 @@ pub fn account_rotate_authentication_key(new_auth_key: Vec<u8>) -> TransactionPa
     ))
 }
 
+pub fn account_transfer(to: AccountAddress, amount: u64) -> TransactionPayload {
```

Comment:
shall we add a unit test in the SDK side for the account transfer?

Reply:
I don't think we need to do that unless someone wants to expose this function to the SDK. Otherwise we'd be adding tests for each entry point we add. I do think it might be a good behavior at some point in the near future to at least add tests for each entry point to either the transaction test harness or e2e testsuite.